### PR TITLE
updated contract compiler version to v0.8.0 & removed unused dependencies from package.json

### PR DESCRIPTION
**Pull request by adi44**
Removed Safemath.sol
Changed "now" keyword to "block.timestamp"
Changed path for oppenzeppelin/merkle.js
removed pragma ABI encoder
Changed openzeppelin-contracts to @openzeppelin/contracts_0.4.0

### CODE REVIEW

File: contracts/Bridge.sol [hunk 1]
```diff
@@ -8,7 +8,7 @@ pragma solidity 0.6.11;
 // import "./IVoteManager.sol";
 // import "./JobStorage.sol";
 // import "./IStateManager.sol";
-// import "openzeppelin-solidity/contracts/math/SafeMath.sol";
```
Comment (dev1644) [inline contracts/Bridge.sol:11]: Since we won't be needing `SafeMath` (at v0.8.0+), let's remove this import @adi44

File: package.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
   },
   "homepage": "https://github.com/schelling-network/contracts#readme",
   "dependencies": {
+    "@openzeppelin/contracts": "^4.0.0",
     "@razor-network/merkle": "^0.6.0",
     "jsonfile": "^6.1.0",
     "openzeppelin-solidity": "^3.1.0"
```
Comment (dev1644) [inline package.json:37]: Can we remove `openzeppelin-solidity` dependency since we are not using it anymore? @adi44

**Review by dev1644 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (adi44) [reply package.json]: I have removed it from the package.json @dev1644

File: contracts/Bridge.sol (hunk 1 above)
Comment (adi44) [reply contracts/Bridge.sol:11]: removed this  import line @dev1644

**Comment by dev1644** [comment]
Hi @adi44 , here, we have 9 commits for this PR, changes aren't drastic that much, so it would be better if you just rebase & squash the commits into a single commit with the message `updated  contract compiler version to v0.8.0 & removed unused dependencies from package.json`.

**Review by SamAg19 — APPROVED** [review]

**Review by dev1644 — APPROVED** [review]

**Review by abhishekvispute — APPROVED** [review]

**Review by SamAg19 — APPROVED** [review]

**Comment by SkandaBhat** [comment]
Fixes #73

**Pull request merged**